vowpalwabbit/global_data.h

```diff
@@ -552,6 +559,7 @@ struct vw
   uint32_t holdout_period;
   uint32_t holdout_after;
   size_t check_holdout_every_n_passes;  // default: 1, but search might want to set it higher if you spend multiple passes learning a single policy
+  size_t early_terminate_passes;
```

Comment:
Semantics?

Reply:
This is an option several reductions care about, previously they read it out of the variable map. I moved to exposing the value off all for consumption.

Reply:
I am planning on tackling shared state soon, so I'll leave it as is because I would like to avoid too many duplicate option definitions. Best case would be this is passed as a parameter.